boards/arm/frdm_k64f/Makefile.board

```diff
@@ -4,4 +4,6 @@ DEBUG_SCRIPT = openocd.sh
 OPENOCD_LOAD_CMD = "flash write_image erase ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
 OPENOCD_VERIFY_CMD = "verify_image ${O}/${KERNEL_BIN_NAME} ${CONFIG_FLASH_BASE_ADDRESS}"
 
-export FLASH_SCRIPT OPENOCD_LOAD_CMD OPENOCD_VERIFY_CMD
+JLINK_DEVICE = MK64FN1M0xxx12
```

Comment:
I didn't see a reason to remove the OpenOCD support. I presume we'll update the OpenOCD version in a future SDK? I've also been looking at pyOCD as another option, since that is being actively maintained for NXP devices and is open source.

About documentation, this is something I plan to work on next after these scripts are cleaned up.